lib/token_logic.ex

```diff
@@ -1,11 +1,15 @@
 defmodule GetTokens do
```

Comment:
This file is named `token_logic` but the module is named `GetTokens` – whichever way you choose, I would be consistent. I think it also raises a larger question of code/module organization in general. For me, it kind of feels like `token_logic` and `token_helpers` are two pretty similar modules, which could both be part of one `Token` module? Also, `BestComputerSpot`, `FirstComputerSpot`, and `SpecialComputerMoves` (which is only one move) kind of all smell pretty similar – is there a case for having them as part of one `ComputerMoves` or just a `ComputerPlayer` module? There's not necessarily One Right Answer to any of this, but it's something to think about for when your code will be 1. much bigger and more complicated than this and 2. have to be used/contributed to by other developers.

Reply:
The separation had to do with how Elixir mocks functions when they are in the same module. Mocks are difficult to use when the helper function is in the same module because he module can not be redeclared. The new module was simply to get a boundary layer between token_logic and token_helpers and allow for control of the behavior in testing.

Reply:
Ah gotcha. That makes sense 👍

Reply:
token_ logic has been changed to get_tokens and get_tokens_test respectively

Reply:
All computer moves have been compiled into a single ComputerMoves file.

Reply:
With the addition of a IoFunction module...... token helpers and get tokens was able to be combined to a single module